clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -307,6 +307,10 @@ private Exception invokePartitionsAssigned(final Set<TopicPartition> assignedPar
 
     private Exception invokePartitionsRevoked(final Set<TopicPartition> revokedPartitions) {
         log.info("Revoke previously assigned partitions {}", Utils.join(revokedPartitions, ", "));
+        Set<TopicPartition> revokePausedPartitions = subscriptions.pausedPartitions();
+        revokePausedPartitions.retainAll(revokedPartitions);
+        if (!revokePausedPartitions.isEmpty())
+            log.info("Revoke previously paused partitions {}", Utils.join(revokePausedPartitions, ", "));
```

Comment:
I'm afraid that users might get confused, because it looks like we revoke twice:
```
Revoke previously assigned partitions tp1, tp2.
Revoke previously paused partitions tp1
```

How about it:
`log.info("The pause flag in partitions [{}] will be removed due to revocation.", Utils.join(revokePausedPartitions, ", "));`
WDYT?

Reply:
I agree with you